services/datamanager/client.go

```diff
@@ -122,7 +122,7 @@ func ConvertImageToBytes(image image.Image, mimeType datasyncpb.MimeType) ([]byt
 			return nil, err
 		}
 		imgBytes = buf.Bytes()
-	case datasyncpb.MimeType_MIME_TYPE_UNSPECIFIED, datasyncpb.MimeType_MIME_TYPE_APPLICATION_PCD:
+	case datasyncpb.MimeType_MIME_TYPE_UNSPECIFIED, datasyncpb.MimeType_MIME_TYPE_VIDEO_MP4, datasyncpb.MimeType_MIME_TYPE_APPLICATION_PCD:
```

Comment:
I'm not really sure why we have separate cases here and then immediately fallthrough. It seems like the default cause would handle these MIME types?

However, your change makes sense in the context of continuing this pattern, so let's go for it haha.